File: mobile_client/lib/models/ram.dart
import 'dart:convert';

List<Ram> ramsFromJson(String str) => List<Ram>.from(json.decode(str).map((x) => Ram.fromJson(x)));

class Ram {
  final String name;
  final int ramCapacity;
  final int memoryRate;

  Ram({required this.name, required this.ramCapacity, required this.memoryRate});

  factory Ram.fromJson(Map<String, dynamic> json) => Ram(
        name: json["name"],
        ramCapacity: json["ramCapacity"],
        memoryRate: json["memoryRate"],
      );

  Map<String, dynamic> toJson() => {
        "name": name,
        "ramCapacity": ramCapacity,
        "memoryRate": memoryRate,
      };
}

File: pwa/PwaClient/Models/Ram.cs
public class Ram
{
    public string Name { get; set; }
    public int RamCapacity { get; set; }
    public int MemoryRate { get; set; }

    // Konstruktor
    public Ram(string name, int ramCapacity, int memoryRate)
    {
        this.Name = name;
        this.RamCapacity = ramCapacity;
        this.MemoryRate = memoryRate;
    }
}